### PR TITLE
Add French list of words.

### PR DESCRIPTION
**Pull request by ajdavidl**
I added a French list of words with the lists from the [wikwik](https://fr.wikwik.org/touslesmots.htm) and [listesdemots](https://www.listesdemots.net/touslesmots.htm) sites.

I also changed src/WordLists.jl and test/runtests.jl to support French language.

Finally, I updated the Readme file to include French.

### CODE REVIEW

File: src/WordLists.jl [hunk 1]
```diff
@@ -94,7 +96,11 @@ const LOOKUP_TABLE = Dict(
     "portuguese" => "pt",
     "português" => "pt",
     "por" => "pt",
-    "pt" => "pt"
+    "pt" => "pt",
+	"french" => "fr",
+	"français" => "fr",
+	"fre" => "fr",
+	"fr" => "fr"
```
Comment (LilithHafner) [inline src/WordLists.jl]:
```suggestion
    "french" => "fr",
    "français" => "fr",
    "fre" => "fr",
    "fra" => "fr",
    "fr" => "fr"
```
Two changes: 4 spaces, not tabs; and "fra" is also a three letter abbreviation for French under [ISO639-2](https://www.loc.gov/standards/iso639-2/php/code_list.php) and I'm not aware of any other language "fra" could refer to so there shouldn't be any ambiguity issues down the line.

File: test/runtests.jl [hunk 2]
```diff
@@ -12,12 +12,14 @@ end
...
     @test words("spanish") isa Vector{String}
     @test words("portuguese") isa Vector{String}
+    @test words("french") isa Vector{String}
 end
 
 @testset "Aliases" begin
     @test words("english") == words("en") == words("eng")
     @test words("spanish") == words("es") == words("spa") == words("español")
     @test words("portuguese") == words("pt") == words("por") == words("português")
+    @test words("french") == words("fr") == words("fre") == words("français")
```
Comment (LilithHafner) [inline test/runtests.jl]:
```suggestion
    @test words("french") == words("fr") == words("fre") == words("fra") == words("français")
```

File: test/runtests.jl [hunk 3]
```diff
@@ -29,10 +31,11 @@ end
     @test length(words("english")) < length(words("english"; all=true))
     @test words("spanish") == words("spanish"; all=true) # handle missing extra.txt
     @test words("portuguese") == words("portuguese"; all=true)
+    @test words("french") == words("french"; all=true)
 end
 
 @testset "Multilingual" begin
-    @test words("en") ⊊ words("english", "spanish", "portuguese")
+    @test words("en") ⊊ words("english", "spanish", "portuguese","french")
```
Comment (LilithHafner) [inline test/runtests.jl]:
```suggestion
    @test words("en") ⊊ words("english", "spanish", "portuguese", "french")
```

File: test/runtests.jl [hunk 4]
```diff
@@ -42,7 +45,7 @@ end
 
 @testset "Content" begin
     @testset "Formatting" begin
-        for langs in [("english",), ("spanish",), ("portuguese",), ("english", "spanish", "portuguese")], all in [false, true]
+		for langs in [("english",), ("spanish",), ("portuguese",), ("french",), ("english", "spanish", "portuguese","french")], all in [false, true]
```
Comment (LilithHafner) [inline test/runtests.jl:48]: whitespace tabs => spaces

File: test/runtests.jl [hunk 5]
```diff
@@ -101,4 +104,18 @@ end
         @test "pear" ∉ pt
         @test "lkfjakljf" ∉ pt
     end
+
+	@testset "French Content" begin
```
Comment (LilithHafner) [inline test/runtests.jl:108]: also tabs here (see https://docs.julialang.org/en/v1/manual/style-guide/#Indentation)

**Review by LilithHafner — COMMENTED** [review]
Thank you for your continued contributions! I have some minor comments about the code changes, but the words look great! They include many names (proper nouns), but they appear to mostly be capitalized which makes it easy for end users to filter them out of they prefer to exclude them.

**Comment by ajdavidl** [comment]
Sorry for the tab mistakes.

**Pull request merged**